### PR TITLE
add missing parameter to constructor

### PR DESCRIPTION
**Pull request by schiessle**
inject l10n

Easy fix... please review @DeepDiver1975 @PVince81 @nickvergessen @th3fallen

### CODE REVIEW

**Comment by nickvergessen** [comment]
Ohh

I think we shall write a test to cover this via:
```php
 	public function decrypt($data) {
 		if (empty($this->fileKey)) {
```

**Comment by schiesbn** [comment]
@nickvergessen yes, but this is out of scope for this PR :wink: It is probably not the only place where a additional unit test would be useful.

**Comment by schiesbn** [comment]
Btw: Do you want to write one? :wink:

**Comment by nickvergessen** [comment]
yes, because there are more problems in this

**Comment by nickvergessen** [comment]
Will rebase after jenkins reports back

**Comment by nickvergessen** [comment]
failed on rebase...

File: apps/encryption/tests/lib/crypto/encryptionTest.php [hunk 1]
```diff
@@ -220,7 +231,6 @@ public function dataTestUpdate() {
 		);
 	}
 
-	/**
```
Comment (nickvergessen) [inline apps/encryption/tests/lib/crypto/encryptionTest.php]: hmpf :space_invader:

**Comment by nickvergessen** [comment]
Should be good now @schiesbn

**Comment by schiesbn** [comment]
@nickvergessen thanks for adding the tests :+1:

**Comment by nickvergessen** [comment]
:+1: for not my code

**Pull request merged**